src/components/course-listing.cjsx

```diff
@@ -5,6 +5,7 @@ Router = require 'react-router'
 WindowHelpers = require '../helpers/window'
 
 {CourseListingActions, CourseListingStore} = require '../flux/course-listing'
+{CourseStore} = require '../flux/course'
 {RefreshButton} = require 'openstax-react-components'
```

Comment:
I thought that we were going to make a `allCoursesWithRoles` or something and use that here.  That way the picker will not show for courses where the period is archived.  Or is this addressing a different bug?